includes/managers/widgets.php

```diff
@@ -100,18 +101,22 @@ private function init_widgets() {
 			'share-buttons',
 		];
 
+		$atomic_widgets_filenames = [
```

Comment:
should we need it here? as it's also defined in `register_atomic_widgets`

Reply:
Removed.